charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -16,6 +16,7 @@ data:
       leaderElectionNamespace: {{ .Release.Namespace }}
       workerThreads: {{ .Values.controller.driverWorkerThreads }}
       waitForLunEnabled: {{ .Values.controller.waitForLunEnabled}}
+      volumeAttachRetryLimit: {{ .Values.controller.volumeAttachRetryLimit }}
```

Comment:
With this change, helm chart package needs to be regenerated and helm index needs to be updated.
`helm package charts/latest-v2/azuredisk-csi-driver -d charts/latest-v2`
`helm repo index ./charts --url https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/charts`

Reply:
Oh yeah! Thanks for the reminder.